android/src/main/java/nl/xguard/flic2/util/SisAction.java

```diff
@@ -0,0 +1,10 @@
+package nl.xguard.flic2.util;
```

Comment:
zijn deze bestanden nodig, zo ja, kunnen ze gehernoemd worden naar een verwijzing zonder Sis in de bestandsnaam?

Reply:
SisAction heb ik verwijderd, deze werd enkel gebruikt voor het permission deel

Reply:
👍